mjolnir/core/ConstraintForceField.hpp

```diff
@@ -0,0 +1,76 @@
+#ifndef MJOLNIR_INTERACTION_CONSTRAINT_INTERACTION_HPP
+#define MJOLNIR_INTERACTION_CONSTRAINT_INTERACTION_HPP
```

Comment:
Mjolnir formats an include guard as `MJOLNIR_${path}_${filename}_HPP`.
```suggestion
#ifndef MJOLNIR_CORE_CONSTRAINT_FORCE_FIELD_HPP
#define MJOLNIR_CORE_CONSTRAINT_FORCE_FIELD_HPP
```